gradle.properties

```diff
@@ -19,4 +19,5 @@ org.gradle.daemon=true
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+org.gradle.jvmargs=-Xmx1024m
```

Comment:
revert

Reply:
@immadisairaj 
I have reverted it back.
But i am getting additional scroll fix codes.
I dont kno whether these are changed codes or previous codes.